src/Zoomba/APILibrary.py

```diff
@@ -159,31 +162,32 @@ def validate_response_contains_expected_response(self, json_actual_response, exp
         elif type(actual_response_dict) is list and actual_response_dict:
             if full_list_validation:
                 if actual_response_dict == expected_response_dict:
-                    assert True
+                    return
                 else:
                     for actual_item, expected_item in zip(actual_response_dict, expected_response_dict):
                         self.key_by_key_validator(actual_item, expected_item, ignored_keys, unmatched_keys_list,
                                                   **kwargs)
-                    if len(unmatched_keys_list) > 0:
```

Comment:
Per discussion: 

This section was meant to add visibility to a user by providing the entire response object when full list validation is being used. 

The generate unmatched keys does not provide object level visibility - only key level (Key x is wrong, expected something, actual something else) so a user could see multiple of the same key being listed without knowledge of their proper location in the object.

Reply:
This has been adjusted